packages/app/src/app/pages/SandpackSecret/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect } from 'react';
```

Comment:
Pop-up to get a read-only sandpack-token from a team id and send it back to the parent window.